src/agent/agent.c

```diff
@@ -606,6 +606,74 @@ void agent_set_systemd_user(Agent *agent, bool systemd_user) {
         agent->systemd_user = systemd_user;
 }
 
+static bool ensure_orch_address(Agent *agent) {
```

Comment:
Why was the method moved when it's not called from some new place?

Reply:
Initially, I used `ensure_orch_address` instead of `agent_set_orch_address` and, therefore, had to move it up in the file. Since the used function changed, I moved it back. Good catch!